src/main/java/com/google/cloud/genomics/dataflow/pipelines/VariantSimilarity.java

```diff
@@ -46,21 +55,29 @@
  */
 public class VariantSimilarity {
   private static final String VARIANT_FIELDS
-      = "nextPageToken,variants(start,calls(genotype,callSetName))";
+    = "nextPageToken,variants(start,calls(genotype,callSetName))";
+
+  public static interface VariantSimilarityOptions extends GenomicsDatasetOptions, GCSOptions {
+    @Default.Boolean(true)
+    Boolean getUseAlpn();
```

Comment:
FYI, you don't actually need this option anymore.  I have it removed in my current CLs.  The Dataflow team added ALPN straight onto the worker, so you don't need the work around anymore! :)